pkg/products/rhssocommon/reconciler.go

```diff
@@ -280,9 +269,56 @@ func (r *Reconciler) SetupOpenshiftIDP(ctx context.Context, serverClient k8sclie
 			},
 		})
 	}
+
+	return nil
+}
+
+// Why do we need to do this?
```

Comment:
Same with the comment - should it be removed?

Reply:
Added proper comment